### ISSUE TITLE
itt a kurva kód(választ mindenkitől)

### ISSUE THREAD

**Issue by maraczibence** [maraczibence/cig-nyok#2]
body {
    font-family: Arial, sans-serif;
    margin: 0;
    padding: 0;
}

/* Navbar formázás */
header {
    background-color: #333;
    color: #fff;
    padding: 10px 0;
}

nav ul {
    list-style-type: none;
    margin: 0;
    padding: 0;
}

nav ul li {
    display: inline;
    margin-right: 20px;
}

nav ul li a {
    color: #fff;
    text-decoration: none;
}

nav ul li a:hover {
    text-decoration: underline;
}

/* Tartalom formázás */
main {
    padding: 20px;
}

section {
    margin-bottom: 20px;
}

h2 {
    color: #333;
}

p {
    color: #666;
}

/* lábléc formázás */
footer {
    background-color: #333;
    color: #fff;
    text-align: center;
    padding: 10px 0;
}
ez volt a css
html:
<!DOCTYPE html>
<html lang="hu">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Trabant</title>
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
    <link rel="stylesheet" href="stilus.css">
</head>
<body>
    <header>
        <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
            <a class="navbar-brand" href="#">Trabant</a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
                <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="navbarNav">
                <ul class="navbar-nav">
                    <li class="nav-item">
                        <a class="nav-link" href="#kezdooldal">Kezdőoldal</a>
                    </li>
                    <li class="nav-item">
                        <a class="nav-link" href="#tortenet">Történet</a>
                    </li>
                    <li class="nav-item">
                        <a class="nav-link" href="#jellemzes">Jellemzés</a>
                    </li>
                    <li class="nav-item">
                        <a class="nav-link" href="#technikai">Technikai adatok</a>
                    </li>
                </ul>
            </div>
        </nav>
    </header>
    <main class="container mt-5">
        <section id="kezdooldal">
            <h2>Trabant - Az Ikonikus Kelet-Német Autó</h2>
            <p>A Trabant egy ikonikus kelet-német autómárka, amely a második világháború után, a hidegháború idején vált népszerűvé.</p>
        </section>
        <section id="tortenet">
            <h2>A Trabant Története</h2>
            <p>A Trabant gyártása 1957-ben kezdődött az akkori NDK-ban. Az autót a kelet-német autógyártás egyik jelképeként tartják számon.</p>
        </section>
        <section id="jellemzes">
            <h2>Trabant Jellemzők</h2>
            <p>A Trabant egy kis méretű, megbízható autó volt, amelyet több mint három évtizeden át gyártottak.</p>
        </section>
        <section id="technikai">
            <h2>Technikai Adatok</h2>
            <p>A Trabantot egy kis hengerűrtartalmú, kéthengeres motor hajtotta, ami meglehetősen alapvető volt, de hatékonynak bizonyult az idők során.</p>
        </section>
    </main>
    <footer class="bg-dark text-white text-center py-3">
        <p>&copy; 2024 Trabant Rajongók Klubja</p>
    </footer>

    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
</body>
</html>

**Comment by Cunciii** [comment]
nekem jo

**Comment by grsalexx** [comment]
nekem is
